pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -73,6 +73,11 @@ func (f OperatorFlags) GetSimpleBoolean(key string) bool {
 	return false
 }
 
+func (f OperatorFlags) Get(key string) (val string, exists bool) {
+	v, ext := f[key]
+	return v, ext
+}
```

Comment:
So it is not just flags anymore... We use it as a key/value now?

Reply:
This is used for two purposes:

1. Where I want to check for a null value (because explicit true adds, explicit false removes, incorrect or undefined does nothing as per the feature flags doc)
2. Defining the pull spec for the container, which we want to avoid bundling into the RP (e.g. to avoid situations such as the fluentbit version situation).

(In general: The lack of schema is to avoid backwards/forwards compatibility issues between the RP, Operator, etc, so unfortunately encoding something that limits the uses so we don't misuse them more difficult.)